products/sle/main.pm

```diff
@@ -794,29 +824,9 @@ elsif (get_var("VIRT_AUTOTEST")) {
         loadtest "virt_autotest/reboot_and_wait_up_normal" if get_var('REPO_0_TO_INSTALL');
         loadtest "virt_autotest/download_guest_assets"     if get_var("SKIP_GUEST_INSTALL") && is_x86_64;
     }
-    if (get_var("VIRT_UEFI_GUEST_INSTALL")) {
-        loadtest "virt_autotest/uefi_guest_installation";
-        loadtest "virt_autotest/set_config_as_glue";
-        loadtest "virt_autotest/uefi_guest_verification";
-    }
     if (get_var("VIRT_PRJ1_GUEST_INSTALL")) {
```

Comment:
Have you added 'VIRT_PRJ1_GUEST_INSTALL' in your uefi test suite? your verification runs are not available for now, so I am unable to figure it out.

Reply:
VIRT_PRJ1_GUEST_INSTALL will be added into uefi test suite later. It will have VIRT_PRJ1_GUEST_INSTALL.

Reply:
then uefi test suites have both VIRT_PRJ1_GUEST_INSTALL and VIRT_UEFI_GUEST_INSTALL. if so, should the if condition in load_virt_guest_install_tests() be changed?

Reply:
How to change ? Current if...else condition works. Traditional prj1 does not have VIRT_UEFI_GUEST_INSTALL. And in order to confine the load_virt_guest_install_tests to specific area, there is also VIRT_PRJ1_GUEST_INSTALL in else branch.

Reply:
how about not add VIRT_PRJ1_GUEST_INSTALL to uefi test suties, but add VIRT_UEFI_GUEST_INSTALL here "if (get_var("VIRT_PRJ1_GUEST_INSTALL") || (get_var("VIRT_UEFI_GUEST_INSTALL")", then the if condition in load_virt_guest_install_tests would be clearer.

Reply:
or remove 'get_var("VIRT_PRJ1_GUEST_INSTALL")' in the elsif condition(like else()) in load_virt_guest_install_tests() as it is redundant.

Reply:
exec_and_insert_password in lib/utils.pm use VIRT_PRJ1_GUEST_INSTALL as condition, so I think it would be better to have it in uefi test suite let alone if any other stuff depends on VIRT_PRJ1_GUEST_INSTALL.  I removed VIRT_PRJ1_GUEST_INSTALL in else branch in load_virt_guest_install_tests after thinking twice. Done.